adapt build system for Scala 2.13

mberndt123 opened the pull request:
Hey,

With these changes it's possible to publish the library for both 2.12 and 2.13. I've removed Scala 2.11 because I couldn't make all three versions work with the same build definition, but 2.11 is obsolete at this point so I hope it's OK.

alexknvl commented:
I'll take a look this evening. I usually publish to bintray since I still don't have sonatype credentials (😊), does the plugin still work, I see it is commented out?

mberndt123 commented:
I commented it out because it wasn't working with sbt 1.0 and everything important (compile, test and publishLocal) was working fine. Apparently it was renamed from bintray-sbt to sbt-bintray and moved to another org, so I added that plugin instead. I hope it does more or less the same thing.

mberndt123 commented:
Cool, thanks for merging! Are you going to publish 0.4.0 for Scala 2.13?
